### PR TITLE
What's new in 8.17

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves https://github.com/elastic/security-docs/issues/6237.

Preview: [What’s new in 8.17](https://security-docs_bk_6286.docs-preview.app.elstc.co/guide/en/security/8.x/whats-new.html)

### CODE REVIEW

File: docs/whats-new.asciidoc
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:25]:
```suggestion
<<alert-suppression,Alert suppression>> now supports the EQL sequence rule type. You can use it to reduce the number of repeated or duplicate detection alerts generated from EQL sequence rules.
```

File: docs/whats-new.asciidoc
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:19]: https://github.com/elastic/security-docs/pull/6272 needs to be merged for this reference to properly resolve. 

```suggestion
The {ref}/logs-data-stream.html[logsDB index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact>> to learn how it can impact your rules and alerts. 
```

**Comment by natasha-moore-elastic** [comment]
CI checks will fail until https://github.com/elastic/security-docs/pull/6272 is merged.

File: docs/whats-new.asciidoc
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:36]: Minor correction, based on the naming used in the [feature docs](https://www.elastic.co/guide/en/kibana/current/cases-action-type.html).
```suggestion
The {kibana-ref}/cases-action-type.html[Cases action] feature, first introduced in 8.14, is moving from technical preview to general availability. Use this action to automatically create cases from rules.
```

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for pulling these together at the last minute!

File: docs/whats-new.asciidoc
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:19]: Re-adding the link so you can merge this comment after I merge https://github.com/elastic/security-docs/pull/6272.

```suggestion
The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact,Using logsdb index mode with {elastic-sec}>> to learn how it can impact your rules and alerts.

NOTE: To use the {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`] feature, you must have the appropriate subscription. Refer to the subscription page for https://www.elastic.co/subscriptions/cloud[Elastic Cloud] and {subscriptions}[Elastic Stack/self-managed] for the breakdown of available features and their associated subscription tiers.
```

File: docs/whats-new.asciidoc
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:34]:
```suggestion
== Cases action is generally available
```

**Review by nastasha-solomon — APPROVED** [review]

**Comment by natasha-moore-elastic** [comment]
@Mergifyio backport 8.17

**Pull request merged**
